modules/cnv-vm-create-nic-web.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
```

Comment:
Didn't test the contents of this file, due to issue in accessing web UI.
I  configured the NIC via CLI ("oc exec... ip link add...").